test/sbm.jl

```diff
@@ -0,0 +1,53 @@
+using Base.Test
+using LightGraphs
+import LightGraphs: nearbipartiteSBM, sample, StochasticBlockModel, graph, blockfractions, blockcounts
+#using ArgParse
+using PyPlot
+
+function generate_nbp_sbm(numedges, sizes)
+    density = 1
+    #= print(STDERR, "Generating communites with sizes: $sizes\n") =#
```

Comment:
Sorry, this is such a minor nit, but could you use `# ` (hash + whitespace) for comments? It makes it easier to search code for them.